src/layers/BitmapLayer.js

```diff
@@ -90,8 +110,19 @@ export default class BitmapLayer extends BaseBitmapLayer {
     });
   }
 }
+export default class BitmapLayer extends CompositeLayer {
+  renderLayers() {
+    const { photometricInterpretation } = this.props;
+    const transparentColor = getTransparentColor(photometricInterpretation);
+    return new BitmapLayerWrapper(this.props, {
+      transparentColor,
+      id: `${this.props.id}-wrapped`
```

Comment:
This ID probably isn't very informative. Is it necessary?

Reply:
Yes, we need to give it a unique id.  Open to other suggestions.  `${this.props.id}-bitmap`?

Reply:
How is that more unique than `this.props.id`, which will be passed down to `BitmapLayerWrapper`?

Reply:
The exported `BitmapLayer` gets the id passed to it, but then `BitmapLayerWrapper` also needs a unique id.  This paradigm plays out in other parts of the repo, but maybe I am missing something here.  I got the usual error about ids without this line.

Reply:
Ok so the `BitmapLayerWrapper` instance needs a unique id from the `BitmapLayer`. Makes sense. I guess this is probably the best way to inject this prop in one place.